core/kernel/src/feature_manifest.rs

```diff
@@ -0,0 +1,270 @@
+use serde::Deserialize;
+use std::collections::HashMap;
+
+#[derive(Deserialize, PartialEq, Debug)]
```

Comment:
Are we going to use partial equivalence on manifests?

Reply:
Used for equality check tests.

Reply:
Right, but that's only in tests, isn't it?

Reply:
Yes